QUEST_LV_0200.tsv

```diff
@@ -956,9 +956,9 @@ QUEST_LV_0200_20150317_000955	{memo X}Give me the herbs.
 QUEST_LV_0200_20150317_000956	{memo X}I need herbs to make the medicine.
 QUEST_LV_0200_20150317_000957	{memo X}Completed preparing medicine.
 QUEST_LV_0200_20150317_000958	{memo X}You need to get all of Mayvern's keys first. {nl}And you need to solve a lot of secrets to get the key.
-QUEST_LV_0200_20150317_000959	Unfortunately, it is Maven's will that you solve the secret by yourself. {nl}First, go to the platform in the Memile Oratorium to solve the first secret and get the holy relic of mercy.
-QUEST_LV_0200_20150317_000960	Writings on the platform
-QUEST_LV_0200_20150317_000961	Remember the holy number with your two eyes. {nl}The light of candle drives away the evil dark.
+QUEST_LV_0200_20150317_000959	$Unfortunately, it is Maven's will that you solve every secret by yourself. {nl}First, get on the platform at the Meile Oratorium to solve the first secret and get the Holy Relic of Mercy.
+QUEST_LV_0200_20150317_000960	$Writing on the Platform
+QUEST_LV_0200_20150317_000961	$Remember the holy number with your two eyes. {nl}The light of candles drive away the evil dark.
```

Comment:
symbolism: light vs evil. if we only had "darkness" well, then the line would sound more practical than it really is.